lib/tasks/evm_application.rb

```diff
@@ -159,25 +159,6 @@ def self.workers_status(servers)
     data.sort_by { |w| [w["Region"], w["Zone"], w["Server"], w["Type"], w["PID"]] }
   end
 
-  def self.update_start
-    require 'fileutils'
-    filename = MiqServer.pidfile
```

Comment:
Do we still need the pidfile code?

Reply:
Looks like when the server starts in `lib/workers/evm_server.rb` it uses it to create the pidfile: `PidFile.create(MiqServer.pidfile)`
Not sure where else we use that pid file but I assume it is still used

Reply:
I followed this a little bit and `MiqServer.running?` still needs it, which is, in turn, called by `EvmApplication.start` to see if a server is already running before starting.  However, as we move towards systemd, I'm not sure that's needed.  At that point I think it would only be needed for local dev, but there might be a simpler way of doing that.